back/lib/shared/hello/hello.go

```diff
@@ -0,0 +1,5 @@
+package hello
+
+func HelloWorld() (string, int) {
+	return "HelloWorld", 25
+}
```

Comment:
If this is a template test or an example, it should be moved elsewhere, if it's not to be kept, then why has it been tracked?